src/identity.js

```diff
@@ -594,7 +594,12 @@ export class Identity extends EventEmitter {
      * @param {object} options
      * @param {string} [options.acrValues] - Authentication Context Class Reference Values. If
      * omitted, the user will be asked to authenticate using username+password. 'otp-email' means
-     * one time password using email. 'otp-sms' means one time password using sms
+     * one time password using email. 'otp-sms' means one time password using sms.
+     * For 2FA (Two-Factor Authentication) possible values are `sms`, `otp` (one time password) and
```

Comment:
Suggestion: Move this new stuff about 2FA before the old `otp-email`/`otp-sms` values.